### DISCUSSION TITLE
Welcome to conserver Discussions!

### DISCUSSION THREAD

**Discussion by bstansell** [bstansell/conserver#49] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bstansell** [comment, marked as answer]
I'm planning on replacing the mailing lists (https://www.conserver.com/pipermail/users/) with this Discussions interface.  Should be better!

**Comment by emaste** [comment]
Hi there, I'm happy with mailing lists, but this is fine too :+1:

I'm interested in all things FreeBSD-related, and use conserver in FreeBSD developer labs.
